docs/docs-content/clusters/edge/trusted-boot/deployment-day2/upgrade-cluster.md

```diff
@@ -79,15 +146,15 @@ not be able to access any data in the persistent partitions.
    You should also receive an output from the build command that contains the OS layer of your cluster profile. Copy and
    save it during the later steps.
 
-5. Issue the following command to push the image to the repository you specified in **.arg**. Replace `IMAGE-TAG` with
+9. Issue the following command to push the image to the repository you specified in **.arg**. Replace `IMAGE-TAG` with
```

Comment:
One minor note to callout to the reader. Depending on what repository they used, they may have to log in to their repository through the repo's preferred authentication method.